src/oas3/transformers/request.ts

```diff
@@ -39,8 +39,13 @@ export function translateParameterObject(
   document: DeepPartial<OpenAPIObject>,
   parameterObject: ParameterObject,
 ): IHttpParam | any {
+  const examples = map(parameterObject.examples, (example, key) => ({
+    key,
+    ...example,
+  }));
```

Comment:
OpenAPI `examples` property is key/value pairs, where the key is used by people for different things (sometimes to name the example), and the value is the the example itself.

Reply:
Aaaah, then I'm double confused. First I wanted to say that with any constant-name given to the `example`, I'd be worried about possible key collisions. Then I deleted that because I thought this was an array-map.

In that case, we should handle possible collisions. If we drop the `example` in case `examples` already has a `default` that's fine by me, but let's not break the guarantee that keys are unique.

Reply:
Dropping `example` in case of collision is fine by me - this should be exceedingly rare edge case.

Reply:
I did what you wrote here, but it sounds kind of strange to me - if there is a "default" example on examples list, we will have no chance to retrieve the singular example. 🤔

Reply:
@mpodlasin that's true, but a more permissive approach (that still retains key-uniqueness) sounded too hard. If you have a good idea, go for it.